package.json

```diff
@@ -1,43 +1,28 @@
 {
   "name": "create",
-  "version": "1.0.0",
+  "version": "3.0.0",
   "description": "",
   "main": "index.js",
   "scripts": {
     "test": "NODE_ENV=testing jest __tests__",
     "build": "ncc build index.js --minify",
-    "watch": "ncc build index.js --watch --minify"
+    "start": "ncc build index.js --watch --minify"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@actions/core": "^1.0.0",
-    "lodash": "^4.17.11",
-    "node-fetch": "^2.3.0",
+    "@actions/core": "^1.10.0",
+    "lodash": "^4.17.21",
+    "node-fetch": "^3.3.0",
```

Comment:
FYI, node-fetch v3 is a [ESM-only module](https://github.com/node-fetch/node-fetch#commonjs). Usually, the project should have `"type": "module"` in package.json to use ESM modules. But maybe ncc bundler can convert it into CJS.

Reply:
yep, already done